api/security.go

```diff
@@ -289,7 +296,14 @@ func CreatePolicy(c *gin.Context) {
 	if !ok {
 		return
 	}
-	response, err := anchore.MakeAnchoreRequest(commonCluster.GetOrganizationId(), commonCluster.GetUID(), http.MethodPost, "policies", policyBundle)
+	anchoreRequest := anchore.AnchoreRequest{
+		OrgID:     commonCluster.GetOrganizationId(),
+		ClusterID: commonCluster.GetUID(),
+		Method:    http.MethodPost,
+		URL:       "policies",
```

Comment:
It may worth factoring out `policies` into a const in case it has to be replaced in the future.